service/waterfall.go

```diff
@@ -637,6 +638,23 @@ func (uis *UIServer) waterfallPage(w http.ResponseWriter, r *http.Request) {
 	}{newUILink, uis.Settings.Jira.Host, uis.GetCommonViewData(w, r, false, true)}, "base", "waterfall.html", "base_angular.html", "menu.html")
 }
 
+// Create and return a redirect to the spruce waterfall page if the user is opted in to the new UI.
+func (uis *UIServer) mainlineCommitsRedirect(w http.ResponseWriter, r *http.Request) {
+	u := gimlet.GetUser(r.Context())
+	if u != nil {
+		user, ok := u.(*user.DBUser)
+		if ok {
+			if user.Settings.UseSpruceOptions.SpruceV1 {
+				evergreen.GetEnvironment().Settings().Ui.UIv2Url = ""
```

Comment:
What's the reason for setting this?

Reply:
I don't remember setting this it must have been a bad copy:pasta when i was cleaning up the code to make the pr. It didn't have an effect which is why i didn't notice it.